.github/workflows/build.yml

```diff
@@ -12,9 +12,7 @@ jobs:
       matrix:
         # Use these Java versions
         java: [
-          1.8,  # Minimum supported by Minecraft
-          11,   # Current Java LTS
-          15    # Latest version
+          16    # Latest version
```

Comment:
Java 16 won't be the latest version in a few months. `Minimum supported by Minecraft` would fit here better.